### PR TITLE
Update variable names in wavernn model

### PR DESCRIPTION
**Pull request by jimchen90**
This is to update the variable names in [WaveRNN model](https://github.com/pytorch/audio/pull/735).

Related to #735

### CODE REVIEW

File: test/test_models.py [hunk 1]
```diff
@@ -65,75 +65,80 @@ def test_waveform(self):
...
         x = torch.rand(n_batch, 1, hop_length * (n_time - kernel_size + 1))
         mels = torch.rand(n_batch, 1, n_freq, n_time)
         out = model(x, mels)
 
-        assert out.size() == (n_batch, 1, hop_length * (n_time - kernel_size + 1), 2 ** n_bits)
+        assert out.size() == (n_batch, 1, hop_length * (n_time - kernel_size + 1), n_classes)
 
     def test_mol(self):
-        """Validate the output dimensions of a _WaveRNN model in mol mode.
+        """Validate the output dimensions of a _WaveRNN model in mol loss.
```
Comment (vincentqb) [inline test/test_models.py:122]: nit: "with mo loss"

File: torchaudio/models/_wavernn.py [hunk 2]
```diff
@@ -205,54 +205,54 @@ class _WaveRNN(nn.Module):
...
-        if self.mode == 'waveform':
-            self.n_classes = 2 ** n_bits
-        elif self.mode == 'mol':
+        if self.loss == 'crossentropy':
+            self.n_classes = n_classes
+        elif self.loss == 'mol':
             self.n_classes = 30
         else:
-            raise ValueError(f"Expected mode: `waveform` or `mol`, but found {self.mode}")
+            raise ValueError(f"Expected loss: `crossentropy` or `mol`, but found {self.loss}")
```
Comment (vincentqb) [inline torchaudio/models/_wavernn.py:252]: We discussed offline that we would remove the "loss/mode" parameter for WaveRNN. Let's do it as part of this PR.

File: torchaudio/models/_wavernn.py [hunk 3]
```diff
@@ -50,8 +50,8 @@ class _MelResNet(nn.Module):
     Args:
         n_res_block: the number of ResBlock in stack (default=10)
         n_freq: the number of bins in a spectrogram (default=128)
-        n_hidden: the number of hidden dimensions (default=128)
-        n_output: the number of output dimensions (default=128)
+        n_hidden_resblock: the number of hidden dimensions of resblock (default=128)
+        n_output_melresnet: the number of output dimensions of melresnet (default=128)
```
Comment (vincentqb) [inline torchaudio/models/_wavernn.py:54]: These are clear as they are for a MelResBlock. Let's keep `n_hidden` and `n_output` here, but add "of resblock" and "of melresnet" in the description as you have done here.

File: torchaudio/models/_wavernn.py (hunk 2 above)
Comment (jimchen90) [reply torchaudio/models/_wavernn.py:252]: Yes, I have removed it. I also removed the `sample_rate` parameter since it is not needed.

File: torchaudio/models/_wavernn.py (hunk 3 above)
Comment (jimchen90) [reply torchaudio/models/_wavernn.py:54]: Yes, I keep them as before.

File: test/test_models.py (hunk 1 above)
Comment (jimchen90) [reply test/test_models.py:122]: This test has been removed.
